  z_axis: 1
  z_sign: 1
  yaw_axis: 0
  yaw_sign: 1
  roll_axis: 2
  roll_sign: -1
  pitch_axis: 3
  pitch_sign: 1
  z_max: 0.5
  yaw_max: 180.
  roll_max: 15.
  pitch_max: 15.
  control_xy: 0
  x_axis: 3
  x_sign: 1
  y_axis: 2
  y_sign: 1
  linear_x_max: 1.
  linear_y_max: 1.
